server/audit_math/suite.py

```diff
@@ -153,13 +152,6 @@ def compute_pvalue(
             )
         )
 
-        # Sometimes the upper_n_w_limit is too extreme, causing illegal 0s.
-        # Check and change the limit when that occurs.
-        if np.isinf(null_logLR(upper_n_w_limit)) or np.isinf(
-            LR_derivative(upper_n_w_limit)
-        ):
-            upper_n_w_limit -= 1
```

Comment:
same q: how do you know it's unreachable?

Reply:
upper_n_w_limit is guaranteed to be a finite number. The only way for it to become infinite is if we do something like divide it by really, really small numbers. If you look in these two functions, that can't ever happen. Again, I worked this out on paper (and with a coding example.